src/components/Footer/footer.tsx

```diff
@@ -1,13 +1,59 @@
 import * as React from "react";
-import "./footer.css";
+// import "./footer.css";
+
+import styled from "styled-components";
+
+const FooterContainer = styled.footer`
+  height: 100px;
+  width: 100vw;
+  background-color: blue;
+  ul {
+    list-style: none;
+    text-align: center;
+    font-family: HelveticaNeue;
+    font-size: 1.2rem;
+    padding: 0;
+  }
+  li {
+    margin: 2vw;
+  }
+  a:link {
+    text-decoration: none;
+    color: black;
+  }
+  a:visited {
+    text-decoration: none;
+    color: black;
+  }
```

Comment:
This can also be written as:
`a{
  &:link, &:visited { 
    text-decoration: none;
    color: black;
  }
}`
You can write any pseudo-class like this by the way, heres a great reference for & usage in scss
https://seesparkbox.com/foundry/how_to_use_ampersands_to_simplifiy_your_sass

Reply:
Thanks!